sdk/servicebus/tests.data.yml

```diff
@@ -8,3 +8,4 @@ extends:
     SDKType: data
     TimeoutInMinutes: 120
```

Comment:
I'd recommend bumping these up to 240 in both files;  the Service Bus suite uses dynamic resources to isolate tests and the management plane responsiveness in the China cloud is very, very slow.